.babelrc

```diff
@@ -1,18 +0,0 @@
-{
-    "plugins": [
-        [
-            "@babel/plugin-transform-runtime",
-            {
-                "helpers": true,
-                "polyfill": false,
-                "regenerator": true,
-                "moduleName": "@babel/runtime"
```

Comment:
@babel/runtime helpers come with heavy core-js deps, I strongly advise not to you it in libraries